src/core/ts/background-scripts/background.js/telemetry/generated/performance.ts

```diff
@@ -105,6 +105,46 @@ export const translationEngineWps = new QuantityMetricType({
   disabled: false,
 });
 
+/**
+ * Total number of words on the original web page.
+ *
+ * Generated from `performance.word_count`.
+ */
+export const wordCount = new QuantityMetricType({
+  category: "performance",
+  name: "word_count",
+  sendInPings: ["custom"],
+  lifetime: "ping",
+  disabled: false,
+});
+
+/**
+ * Total number of visible words on the original web page.
+ *
+ * Generated from `performance.word_count_visible`.
+ */
+export const wordCountVisible = new QuantityMetricType({
+  category: "performance",
+  name: "word_count_visible",
+  sendInPings: ["custom"],
+  lifetime: "ping",
+  disabled: false,
+});
+
+/**
+ * Total number of visible words on the original web page that
+ * were visible in the viewport at the time of "translation button pressed".
+ *
+ * Generated from `performance.word_count_visible_in_viewport`.
+ */
+export const wordCountVisibleInViewport = new QuantityMetricType({
```

Comment:
I would stick to only one visible count. We have a lot of different metrics already and I can't imagine building charts based on both of them. Let's just define it somehow and stick to one implmentaiton.

Reply:
I'll remove the visible counts in this PR and instead include one or both as part of fixing https://github.com/mozilla-extensions/bergamot-browser-extension/issues/186